fenix/app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -122,7 +122,10 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
 
         browserToolbarView.view.addNavigationAction(leadingAction)
 
-        updateToolbarActions(isTablet = resources.getBoolean(R.bool.tablet))
+        this.isTablet = resources.getBoolean(R.bool.tablet)
+        if (this.isTablet) {
+            addTabletActions(requireContext())
+        }
```

Comment:
Sorry drive-by: Do we ever care about the else branch here to remove the tablet actions if it's no longer a tablet? Or is it because this is only ever called on `onViewCreated` that we should expect the toolbar to not have any of the tablet actions?

Reply:
The second. We should expect the toolbar to not have any of the tablet actions.

Reply:
That's actually a good question, I couldn't make the app reach `removeTabletActions()` block. I don't have a real foldable device, that might be the issue. @zmckenney Could you help us understand that block of could, in which case do we expect it to get executed?